packages/commons/src/context/context.ts

```diff
@@ -21,7 +22,9 @@ const globalStore = new AsyncLocalStorage<AppContext>();
 const defaultAppContext: AppContext = {
   authData: {
     userId: "",
-    organizationId: "",
+    // this is a workaround to avoid to change the type
+    // from TenantId to TenantId | undefined
+    organizationId: unsafeBrandId(""),
```

Comment:
👍🏻  this is a valid workaround for me, the `defaultAppContext` should be used only for AsyncLocalStorage initialization.